trade-finance-manager-api/src/v1/content-strings/index.js

```diff
@@ -1,6 +1,133 @@
 const DEAL = {
   ELIGIBILITY_CRITERIA: {
     'BSS/EWCS': {
+      1: {
+        1: {
+          text: 'The Supplier has confirmed in its Supplier Supplementary Declaration that the Supply Contract does not involve agents and the Bank is not aware that any of the information contained within it is inaccurate.',
+          textList: [],
+          group: 'Supply Contract / Transaction criteria',
```

Comment:
not a show-stopper, but you shouldn't need the `group` property

Reply:
I did not notice but because it is being used in old EC, I think might be best to leave it in case we need it in future?